index.bs

```diff
@@ -7063,11 +7059,21 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
         production in the [=local end definition=], with the <code>handle</code>
         property set to |handle id| if it's not null, or omitted otherwise.
 
+    <dt>|value| has a \[[GeneratorState]] [=internal slot=]
+    <dd>Let |remote value| be a [=/map=] matching the <code>script.GeneratorRemoteValue</code>
+        production in the [=local end definition=], with the <code>handle</code>
+        property set to |handle id| if it's not null, or omitted otherwise.
+
     <dt>|value| has an \[[ErrorData]] [=internal slot=]
     <dd>Let |remote value| be a [=/map=] matching the <code>script.ErrorRemoteValue</code>
         production in the [=local end definition=], with the <code>handle</code>
         property set to |handle id| if it's not null, or omitted otherwise.
 
+    <dt>|value| has a \[[ProxyHandler]] [=internal slot=] and a \[[ProxyTarget]] [=internal slot=]
```

Comment:
Is the extra check for the `ProxyTarget` internal slot required? Similar to other object types the `ProxyHandler` should probably be enough, or?

Reply:
I don't think it's currently possible to have one without the other, but checking for both seems reasonable and probably the safest assumption.

Reply:
Sounds fine to me then. Thanks.